### ISSUE TITLE
[Accessibility] Crow sounds are missing subtitle language entries.

### ISSUE THREAD

**Issue by Vaelzan** [Alex-the-666/AlexsMobs#157]
All crow sounds are missing subtitles (crow_idle, crow_hurt).

Alex's Mobs 1.5.0
